archive-dssspell.sh

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
 set -e
 
-
 if [[ -z "$1" ]]; then
   echo "You must provide a name option to name the directory"
```

Comment:
just [diffchecked](https://www.diffchecker.com/hdoXC4pm) against spells-mainnet.
It seems there were some changes introduced in https://github.com/makerdao/spells-goerli/pull/40 that might be ported to spells-mainnet as well (remove L7).
For the current changes, I think `date (YYYY-MM-DD)` should be used instead of `name`, to match spells-mainnet.
cc: @brianmcmichael 

```suggestion
  echo "You must provide a date (YYYY-MM-DD) option to name the directory"
```

Reply:
Ok, so we decided that we will only use a date for the directory name in goerli as well.
I guess I did not port this back since this was not intoduced in the latest spell.
Anyway accepting the suggestion.